source/includes/steps-install-bi-connector.yaml

```diff
@@ -125,22 +112,37 @@ post: |
 
   You can examine the generated ``schema.drdl`` file, and remove any fields or
   tables that are not part of your intended schema.
+
 ---
-title: Start ``mongosqld``
+title: Edit your |bi| configuration file.
 ref: start-mongosqld
 level: 4
+content: |
+  Using your preferred text editor, create a |bi| configuration file.
+  To review the configuration file options, see :ref:`Configuration
+  File <config-format>`.
+
+---
+title: Start ``mongosqld`` as a system service.
+ref: start-mongosqld-service
+level: 4
 action:
-  - heading: "Linux / macOS"
+  - heading: "Linux / macOS using a configuration file"
     language: sh
     copyable: true
     code: |
-      mongosqld --schema schema.drdl --mongo-uri {your.mongohost.com}
-  - heading: "Windows"
+      mongosqld install --config {pathToConfigFile}/mongosqld.conf
+      service mongosql start
```

Comment:
This would be `service` or `systemd`. We use this as a standard for all Linuxes for MongoDB:

- [RHEL](https://docs.mongodb.com/manual/tutorial/install-mongodb-enterprise-on-red-hat/#start-mongodb): `sudo service mongod start`

- [Ubuntu](https://docs.mongodb.com/manual/tutorial/install-mongodb-enterprise-on-ubuntu/#start-mongodb): `sudo service mongod start`

- [Debian](https://docs.mongodb.com/manual/tutorial/install-mongodb-enterprise-on-debian/#start-mongodb): `sudo service mongod start`

- [SUSE](https://docs.mongodb.com/manual/tutorial/install-mongodb-enterprise-on-suse/#start-mongodb): `sudo service mongod start`

- [Amazon](https://docs.mongodb.com/manual/tutorial/install-mongodb-enterprise-on-amazon/#start-mongodb): `sudo service mongod start`